packages/core/src/components/results/Results.less

```diff
@@ -27,7 +35,11 @@
   border-bottom: 2px solid #333;
 }
 
-// If several questions are on same row, scores overlap next question. Hack to prevent it.
 table td .e-result-scorecount {
+  width: 8rem;
```

Comment:
Tässä mennään varmaankin lyhyellä aikavälillä oikeaan suuntaan, mutta se että tyylit riippuu tällä tavalla kokeen rakenteesta on tosi hauras ratkaisu.

Tän pistesarakkeen rendaamista pitäis jossain vaiheessa parantaa.

Reply:
Muutin ratkaisua käyttämään omaa luokkaa multi-line ja rich-text vastausten scoressa, jolloin tätä ei tarvita.